libs/components/src/lib/listbox-option/index.ts

```diff
@@ -0,0 +1,16 @@
+import '../focus';
+import '../icon';
+
+import type { FoundationElementDefinition } from '@microsoft/fast-foundation';
+import { designSystem } from '../../shared/design-system';
+import { ListboxOption } from './listbox-option';
+import { ListboxOptionTemplate as template } from './listbox-option.template';
+import styles from './listbox-option.scss';
+
+export const vividListboxOption = ListboxOption.compose<FoundationElementDefinition>({
+	baseName: 'listbox-option',
```

Comment:
should simply be named option as is also used within select component
```suggestion
	baseName: 'option',
```